ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/ImageCve/ImageCvePage.tsx

```diff
@@ -75,16 +72,21 @@ import { resourceCountByCveSeverityAndStatusFragment } from '../SummaryCards/Cve
 import VulnerabilityStateTabs from '../components/VulnerabilityStateTabs';
 import useVulnerabilityState from '../hooks/useVulnerabilityState';
 
-const workloadCveOverviewCvePath = getOverviewCvesPath({
+const workloadCveOverviewCvePath = getOverviewPagePath('Workload', {
     vulnerabilityState: 'OBSERVED',
     entityTab: 'CVE',
 });
 
 export const imageCveMetadataQuery = gql`
-    ${imageCveMetadataFragment}
```

Comment:
Unfortunately we cannot reuse this fragment unless the BE team declares a graphql interface on the ImageCVECore and future Node/Platform CVECore types, so I just inlined it in the query here.